_stbt/pylint_plugin.py

```diff
@@ -105,6 +99,22 @@ def visit_callfunc(self, node):
                                          args=node.as_string())
 
 
+def _is_callable(node):
+    failed_to_infer = True
+    for inferred in node.infer():
+        # Note that when `infer()` fails it returns `YES` which
+        # returns True to everything (including `callable()`).
+        if inferred != YES:
+            failed_to_infer = False
+            if inferred.callable():
+                return True
+    if failed_to_infer:
+        if (isinstance(node, Call) and
+                _is_function_named(node.func, "functools.partial")):
+            return True
+    return False
```

Comment:
I guess the proper fix would be to add an understanding of `functools.partial` to [brain_functools.py](https://github.com/PyCQA/astroid/blob/master/astroid/brain/brain_functools.py), but we'd still need to upgrade pylint after that.